tutorials/forest-cover-loss-estimation/forest-cover-loss-estimation.js

```diff
@@ -0,0 +1,167 @@
+// Set parameters
+var country = 'Bolivia'; // selected country (e.g. Bolivia)
+var cc = ee.Number(10); // canopy cover percentage
+var pixels = ee.Number(6); // minimum forest size in pixels (approximately 0.5 ha in this example)
+var lossPixels = ee.Number(6); // minimum mapping area for tree loss
+
+print('Country: ', country);
+print('Minimum canopy cover (%):', cc);
+print('Minimum forest area (pixel): ', pixels);
+
+// Use Global Forest Change dataset to estimate forest area in year 2000.
+var gfc2018 = ee.Image('UMD/hansen/global_forest_change_2018_v1_6');
+
+// Select 'treecover2000' and display all trees in yellow.
+var canopyCover = gfc2018.select(['treecover2000']);
+Map.addLayer(canopyCover.updateMask(canopyCover), {
+    palette: ['yellow'],
+    min: 1,
+    max: 100
+}, 'All trees (yellow)');
+
+// Create and display where canopy cover is 10% or more in orange. 
+var canopyCover10 = canopyCover.gte(cc);
+
+Map.addLayer(canopyCover10.updateMask(canopyCover10), {
+    palette: ['orange'],
+    max: 100
+}, '>= min canopy cover % (orange)');
+
+// Create connectedPixelCount() to get contiguous area.
+var contArea = canopyCover10.mask(canopyCover10).connectedPixelCount();
+// Apply the threshold (6 pixels). 
+var minArea = contArea.gte(pixels);
+
+// Display the results in the map. The areas less than the threshold are shown in orange.
+Map.addLayer(minArea, {
+    min: 0,
+    palette: ['orange', 'green']
+}, '>= min canopy cover and area size (green)');
+
+// Depending on the level of zoom, you may need to use the nominal scale.
+var prj = gfc2018.projection();
+var scale = prj.nominalScale();
+Map.addLayer(minArea.reproject(prj.atScale(scale)), {
+    min: 0,
+    palette: ['orange', 'green']
+}, '>= min canopy cover and area size (nominal scale)', false);
+
+// Quantify this forest area for a country.
+// Load country features from Large Scale International Boundary (LSIB) dataset.
+var countries = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017');
+var selected = countries.filter(ee.Filter.eq('country_na', ee.String(country)));
+// Center the map to the selected country.
+Map.centerObject(selected, 12);
+Map.addLayer(selected, '', 'Selected country', false);
+
+var forestArea = minArea.multiply(ee.Image.pixelArea()).divide(10000);
```

Comment:
Instead, should I use the following?
var forestArea = minArea.reproject(prj.atScale(scale)).multiply(ee.Image.pixelArea()).divide(10000);